Libraries/Network/NetInfo.js

```diff
@@ -79,56 +80,51 @@ const _isConnectedSubscriptions = new Map();
  * NetInfo exposes info about online/offline status
  *
  * ```
- * NetInfo.fetch().then((reach) => {
- *   console.log('Initial: ' + reach);
+ * NetInfo.getConnection().then((connection) => {
```

Comment:
I see. `The connection attribute, when getting, returns an object that implements the NetworkInformation interface.` Yeah, I think using `getConnectionInfo` or `getConnectionInformation` would be much better.